col/src/main/java/vct/col/typerules/CoercingRewriter.scala

```diff
@@ -250,6 +253,109 @@ abstract class CoercingRewriter[Pre <: Generation]() extends Rewriter[Pre] with
   override def dispatch(region: ParRegion[Pre]): ParRegion[Post] =
     postCoerce(coerce(preCoerce(region)))
 
+
+
+  def preCoerce(node: Verification[Pre]): Verification[Pre] = node
+  def postCoerce(node: Verification[Pre]): Verification[Post] = rewriteDefault(node)
+  override def dispatch(node: Verification[Pre]): Verification[Rewritten[Pre]] = postCoerce(coerce(preCoerce(node)))
```

Comment:
Could you please try making all the `dispatch`es `final`? This prevents the mistake of overriding it out of habit (and will break compilation of most `Import*` passes: we should replace `dispatch` with `postCoerce` in those passes)

Reply:
Which dispatch should be replaced, the one in CoercingRewriter, or the one in the Import* class?

Reply:
The ones in the Import classes